examples/src/transfer.js

```diff
@@ -0,0 +1,7 @@
+const transferResponse = await rhinofi.transfer({
+  recipientEthAddress: '0x5472cf4f1be2aa6ad27c6f93101f7899ccadbaf7',
```

Comment:
Intended hard coded address?

Reply:
yes, we need an address of another user to send the transfer to

I don't think there is a way to query for other users addresses via the api, so I just picked one which is present on both dev and staging

although it just occurred to me that could use the exchange address instead which is available on the config

Reply:
fixed in 6595f04f4fff0e7506110dcc92813bd5f0b61f90